### PR TITLE
Replace link to tool that doesn't support IPv6

### PR DESCRIPTION
**Pull request by ruurtjan**
#### Summary
This PR replaces a link to a tool that doesn't support IPv6 with a tool that does.

#### Motivation
Many sites now listen to both IPv4 and IPv6 addresses. The MDN docs should promote IPv6 adoption.

I originally made a PR to the `mdn/content` repo, which got merged. But then I noticed that the translated content also contains links to a tool that doesn't support IPv6, and @teoli2003 asked me to create a PR here too. See https://github.com/mdn/content/pull/17468

#### Supporting details
Old link: 
image

New link:
image

#### Related issues

https://github.com/mdn/content/pull/17468

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by irvin — APPROVED** [review]

**Comment by ruurtjan** [comment]
I pushed a commit that also replaces the link in the JA translation.

**Review by marcelozarate — APPROVED** [review]

**Review by Graywolf9 — CHANGES_REQUESTED** [review]
Thank you so much! I let you a comment

File: files/pt-br/learn/getting_started_with_the_web/how_the_web_works/index.html [hunk 1]
```diff
@@ -76,7 +76,7 @@ <h2 id="DNS_explicado">DNS explicado</h2>
 
 <p>Isso é chamado {{Glossary("IP Address", "endereço IP")}} e representa um local único na web. No entanto, não é muito fácil de lembrar, é? É por isso que o Servidor de Nome de Domínio foi inventado. Esses são servidores especiais que relacionam o endereço da web que você digita no seu navegador (como "mozilla.org") com o endereço real do site (IP).</p>
 
-<p>Sites podem ser acessados diretamente pelo seu endereço IP. Você pode encontrar o endereço IP de um site, digitando seu domínio em uma ferramenta como o <a href="https://ipinfo.info/html/ip_checker.php">IP Checker</a>.</p>
+<p>Sites podem ser acessados diretamente pelo seu endereço IP. Você pode encontrar o endereço IP de um site, digitando seu domínio em uma ferramenta como o <a href="https://www.nslookup.io/website-to-ip-lookup/">IP Checker</a>.</p>
```
Comment (nathipg) [inline files/pt-br/learn/getting_started_with_the_web/how_the_web_works/index.html:79]:
```suggestion
<p>Sites podem ser acessados diretamente pelo seu endereço IP. Você pode encontrar o endereço IP de um site, digitando seu domínio em uma ferramenta como o <a href="https://www.nslookup.io/website-to-ip-lookup/">DNS lookup</a>.</p>
```

**Review by nathipg — CHANGES_REQUESTED** [review]
Thanks for the contribution!
I agree with @Graywolf9 about the link text, could you also change it in pt-br, please?

**Comment by ruurtjan** [comment]
Thanks! I've accepted the suggested changes.

**Review by Graywolf9 — APPROVED** [review]

File: files/ja/learn/getting_started_with_the_web/how_the_web_works/index.html [hunk 2]
```diff
@@ -93,7 +93,7 @@ <h2 id="DNS_explained">DNS の説明</h2>
 
 <p>これは、 {{Glossary("IP Address", "IP アドレス")}}と呼ばれ、ウェブ上の一意の場所を表します。しかし、あまり覚えやすくはないですよね。だから、ドメインネームサーバーが発明されたのです。これらは特別なサーバーで、ブラウザーに入力したウェブアドレス ("mozilla.org" など) とウェブサイトの実際の (IP) アドレスを対応させます。</p>
 
-<p>ウェブサイトには、IP アドレスを使用して直接アクセスできます。<a href="https://ipinfo.info/html/ip_checker.php">IP Checker</a> のようなツールにドメインを打ち込むことで、ウェブサイトの IP アドレスを見つけることができます。</p>
+<p>ウェブサイトには、IP アドレスを使用して直接アクセスできます。<a href="https://www.nslookup.io/website-to-ip-lookup/">IP Checker</a> のようなツールにドメインを打ち込むことで、ウェブサイトの IP アドレスを見つけることができます。</p>
```
Comment (mfuji09) [inline files/ja/learn/getting_started_with_the_web/how_the_web_works/index.html:96]: Can you replace the link text?

```suggestion
<p>ウェブサイトには、IP アドレスを使用して直接アクセスできます。<a href="https://www.nslookup.io/website-to-ip-lookup/">DNS lookup</a> のようなツールにドメインを打ち込むことで、ウェブサイトの IP アドレスを見つけることができます。</p>
```
```suggestion
<p>ウェブサイトには、IP アドレスを使用して直接アクセスできます。<a href="https://www.nslookup.io/website-to-ip-lookup/">IP Checker</a> のようなツールにドメインを打ち込むことで、ウェブサイトの IP アドレスを見つけることができます。</p>
```

**Review by mfuji09 — CHANGES_REQUESTED** [review]
Can you replace the link text with the correct site name?

**Comment by ruurtjan** [comment]
@mfuji09: done :)

**Review by mfuji09 — APPROVED** [review]

**Comment by cw118** [comment]
Gently poking @mdn/yari-content-pt-br @mdn/yari-content-ru @mdn/yari-content-ko for reviews :)

**Review by igy95 — APPROVED** [review]
Thank you for your contribution :)

**Review by nathipg — APPROVED** [review]
Great job!

**Pull request merged**